docs/getting-started-in-5-minutes.md

```diff
@@ -116,6 +155,10 @@ This simple smart-contract responds with `PONG` to a `PING` message sent to the
 ```bash
 cd ~/gear/contracts/first-gear-app/
 RUSTFLAGS="-C link-args=--import-memory" cargo +nightly build --release --target=wasm32-unknown-unknown
+
```

Comment:
Need to remove it.
```suggestion
```

Reply:
Yeah, `gear-wasm-builder` puts everything instead of you. So `cargo build` and `cargo test` or `cargo test --release` are the only required commands